packages/eas-cli/src/commands/env/get.ts

```diff
@@ -38,21 +39,24 @@ export default class EnvironmentVariableGet extends EasCommand {
   };
 
   static override flags = {
-    name: Flags.string({
+    'variable-name': Flags.string({
       description: 'Name of the variable',
     }),
+    'variable-environment': Flags.enum<EnvironmentVariableEnvironment>({
+      ...EasEnvironmentFlagParameters,
+      description: 'Current environment of the variable',
+    }),
```

Comment:
what do you think about using the `current-name` and `target-name` flag naming convention?

Reply:
I was thinking about it and I'm opting for `variable-name`. `current-name` makes sense in `env:update` command, but I want it to be consistent across all functions. In `env:get` `current-name` seems off: why is it current? why would I use not-current name? IMO `variable-name` fits both mutations and queries.